source/main.c

```diff
@@ -26,6 +26,7 @@ int main()
     Debug("Y: NAND Padgen");
     Debug("L: NAND Partition Dump");
     Debug("R: NAND Dump");
+    Debug("%c: Ticket Dump", 0x18);
     Debug("");
```

Comment:
It looks better this way (only one symbol, like all other options) - especially if more stuff gets added later later (for right, left, down). If I manage to port more stuff, I think it would be a better option to topically group options, implement a menu switch key, and don't use the arrow keys at all.